src/hipscat/inspection/almanac.py

```diff
@@ -0,0 +1,175 @@
+import glob
+import os
+from typing import List
+
+import pandas as pd
+
+from hipscat.catalog.catalog import CatalogType
+from hipscat.catalog.dataset.dataset import Dataset
+from hipscat.inspection.almanac_catalog_info import AlmanacCatalogInfo
+
+
+class Almanac:
```

Comment:
How would you get one catalog from another? E.g. if you had an object catalog and you wanted to get the source catalog for it. Is there some way you can use the catalog object you have, or would you have to use the full name of the other catalog?